### PR TITLE
fix: biometric storage build issue

### PR DESCRIPTION
**Pull request by sachins-geekyants**
**- What I did**
- adding fix for android build issue
https://github.com/atsign-foundation/apps/issues/721
https://github.com/atsign-foundation/apps/issues/720

**- How I did it**
- upgraded `biometric_storage` to latest version
- updated readme for usage on windows platform.

**- How to verify it**
- run android by overriding `at_client_mobile` with this branch.
- run windows by overriding `at_client_mobile` with this branch and also updating dependencies as mentioned in readme.

**- Description for the changelog**

### CODE REVIEW

**Comment by cconstab** [comment]
Is this the only solution  for windows? Seem a kludge to me..

File: at_client_mobile/README.md [hunk 1]
```diff
@@ -136,6 +136,13 @@ flutter:
     - .env
 ```
 
+- If your app supports windows platform then add `biometric_storage` in app's dependencies
```
Comment (murali-shris) [inline at_client_mobile/README.md:139]: we are adding biometric_storage in pubspec..this dependency will be fetched for all platforms..right?

Comment (sachins-geekyants) [reply at_client_mobile/README.md:139]: It throws following exception, on windows platform if does not added under dependencies section.

```
flutter: #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:294:7)
<asynchronous suspension>
#1      MethodChannelBiometricStorage.getStorage (package:biometric_storage/src/biometric_storage.dart:318:22)
<asynchronous suspension>
#2      KeyChainManager.readAtClientData (package:at_client_mobile/src/keychain_manager.dart:35:21)
<asynchronous suspension>
#3      KeyChainManager.disableUsingSharedStorage (package:at_client_mobile/src/keychain_manager.dart:84:18)
<asynchronous suspension>
#4      KeyChainManager.initialSetup (package:at_client_mobile/src/keychain_manager.dart:78:7)
<asynchronous suspension>
#5      OnboardingService.initialSetup (package:at_onboarding_flutter/services/onboarding_service.dart:87:5)
<asynchronous suspension>
#6      _AtOnboardingStartScreenState._init (package:at_onboarding_flutter/screen/at_onboarding_start_screen.dart:37:5)
<asynchronous suspension>

```

**Comment by sachins-geekyants** [comment]
I have raised a ticket in `biometric_storage` repo regarding windows strange behaviour (https://github.com/authpass/biometric_storage/issues/85)

**Review by murali-shris — APPROVED** [review]

**Pull request merged**